packages/engine/src/scene/components/ModelComponent.tsx

```diff
@@ -104,14 +104,6 @@ function ModelReactor() {
 
   const [gltf, error] = useGLTF(modelComponent.src.value, entity)
 
-  useEffect(() => {
```

Comment:
This was added to prevent an infinite loading screen if a model component existed in the scene without a source value (since the scene is never loaded), if we don't want to add an error for that case we may need to create a dummy scene if there's no source value

Reply:
So would that just be returning an empty scene when passing a blank URL into useGLTF?

Reply:
We already generate a dud Group if the gltf is blank

Reply:
The GLTFComponent requires that all ModelComponent instances either have a valid ModelComponent.scene value or an error before marking a GLTF as loaded, I think this was to prevent models falling through colliders initially